File: src/main/java/frc/utils/rumble/Rumble.java
package frc.utils.rumble;

public class Rumble{
    public double powR;
    public double powL;
    public double time;
    public Rumble(double time, double pow){
        this.powR = pow;
        this.powL = pow;
        this.time = time;
    }
    public Rumble(double time, double powR, double powL){
        this.powR = powR;
        this.powL = powL;
        this.time = time;
    }
}

File: src/main/kotlin/frc/utils/rumble/Rumble.kt
package frc.utils.rumble

class Rumble {
    @JvmField var powR: Double
    @JvmField var powL: Double
    @JvmField var time: Double

    constructor(time: Double, pow: Double) {
        this.powR = pow
        this.powL = pow
        this.time = time
    }

    constructor(time: Double, powR: Double, powL: Double) {
        this.powR = powR
        this.powL = powL
        this.time = time
    }
}
